modules/administration-guide/examples/snip_che-upgrading-che-7-41-on-openshift.adoc

```diff
@@ -0,0 +1,6 @@
+Upgrading from {prod-prev-short} {prod-last-version-pre-dwo} to {prod-short} {prod-ver} on OpenShift requires first upgrading to {prod-short} 7.49.
```

Comment:
```suggestion
Upgrading from {prod-prev-short} {prod-last-version-pre-dwo} to {prod-short} {prod-ver} on Red Hat OpenShift Container Platform requires first upgrading to {prod-short} 7.49.
```
Our docs must use the full product name for OpenShift. Do you have an attribute for that?

Reply:
Where are you adding this snip? I don't see an include statement for it.